src/shared/components/common/markdown-textarea.tsx

```diff
@@ -358,7 +359,9 @@ export class MarkdownTextArea extends Component<
       return
     } 
     if (event.target.files.length > maxUploadImages) {
-        toast(i18n.t("too_many_images_upload"), "danger");
+        toast(i18n.t("too_many_images_upload",{
+          maxFormatted: numToSI(maxUploadImages),
```

Comment:
Check the other count examples in this codebase, there are many.

Reply:
It's done. Waiting for https://github.com/LemmyNet/lemmy-translations/pull/28

Reply:
Does it look good?

Reply:
@dessalines Looks fine to me now. Checks were passing. Anything left over here? I hope to get this into the next release.

Reply:
Sorry, just finishing up a few other things, so it might be a few days till I get to this. Don't worry we won't do the 0.17 release without it.

Reply:
@dessalines Alright, no problem. Thanks a lot!